packages/build/src/tarball.spec.ts

```diff
@@ -82,14 +83,47 @@ describe('tarball module', () => {
 
         let accessErr;
         try {
-          fs.access(expectedTarball);
+          await fs.access(expectedTarball);
         } catch (err) {
           accessErr = err;
         }
 
         expect(accessErr).to.be.undefined;
       });
     });
+
```

Comment:
A better way than commenting this out would be this:

```
describe('.tarballRedhat', () => {
    beforeEach(async function() {
        if(!commandExists('rpmbuild')) {
            this.skip();
        }
    });
    
    it('...', () => { });
})
``` 

`this.skip()` skips all the tests in the current test block;

`commandExists` is this thing here: https://www.npmjs.com/package/command-exists

Reply:
oh! didn't know something like this existed. Will change what I currently have!